docs/photoniq/vwrs/configure-vwrs-priority.md

```diff
@@ -0,0 +1,85 @@
+---
+sidebar_position: 30
+title: Configure a VWRs Priority Waiting Room
+---
+
+Priority waiting rooms allow you to prioritize traffic based on the priority of the request. Requests are assigned a priority level and then dequeued at different rates based on the priority level. This feature helps handle various types of traffic, such as high-priority, regular, and low-priority.
```

Comment:
If we have a separate topic for priority waiting rooms, then we need separate topics for the other waiting roo types to be consistent.